staging/src/k8s.io/apiserver/pkg/server/healthz/healthz_test.go

```diff
@@ -177,3 +183,43 @@ func TestFormatQuoted(t *testing.T) {
 		})
 	}
 }
+
+func TestGetExcludedChecks(t *testing.T) {
+	type args struct {
+		r *http.Request
+	}
+	tests := []struct {
+		name string
+		r    *http.Request
+		want sets.String
+	}{
+		{"Should have no excluded health checks",
+			createGetRequestWithUrl("http:localhost:8080/healthz?verbose=true"),
```

Comment:
I'd use `/healthz?verbose=true` instead of `http:localhost:8080/healthz?verbose=true`, since the host name is being parsed wrong (I think?). I was confused about how it could be working until I realized that this isn't actually calling the HandlerFunc. :)

Reply:
oh, nice catch. i'll fix it.